apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/S3Utilities.java

```diff
@@ -16,6 +21,12 @@
    * @return the {@link AmazonS3} client to use
    */
   public static AmazonS3 createS3Client(ExtractionOptions options) {
+    Boolean s3LocalDev = Boolean.parseBoolean(System.getProperty("s3.local", "false"));
```

Comment:
This appears in three places. Perhaps it should be a static method on the S3MinioConfig class?